src/Response/GetPayload/PayloadMeta.php

```diff
@@ -23,6 +23,7 @@ public function __construct(
         public readonly ?bool $immutable = null,
         public readonly ?string $returnUrlApp = null,
         public readonly ?string $returnUrlWeb = null,
+        public readonly ?bool $pathfinding = null,
```

Comment:
I can't find `pathfinding` in the payload metadata docs. Are the docs outdated or is it actually not supported yet @WietseWind?

Reply:
That's a new one, it is in the `options` in the docs:
<img width="1123" alt="image" src="https://user-images.githubusercontent.com/4756161/221715400-7718b4b6-9439-4dd1-ae1a-0b54abe5c739.png">

Reply:
Right, so it's added in the `Options` class, but also in the Payload metadata class here. That's the part I can't find in the docs.

Reply:
That `pathfinding` boolean can be found here in the 200 response docs: https://xumm.readme.io/reference/get-payload

Reply:
Ah I see, I was looking at the custom meta of the create payload payload... payload payload. LGTM then, this was the only thing left on my list.